tests/testthat/test_agg_scale.R

```diff
@@ -694,18 +681,17 @@ test_that(description, {
                      col_stem = "location",
                      col_type = "categorical",
                      mapping = iran_mapping),
-               regexp = "input data is missing in `dt`")
+               regexp = "expected input data is missing")
   expect_error(scale(dt = new_input_dt,
                      id_cols = id_cols,
                      value_cols = value_cols,
                      col_stem = "location",
                      col_type = "categorical",
                      mapping = iran_mapping,
                      collapse_missing = TRUE),
-               regexp = "input data is missing in `dt`")
+               regexp = "expected input data is missing")
```

Comment:
What do you think about adding another test that aggregation doesn't exceed a certain amount of runtime for a small example?

Reply:
Addressed in #55. We could potentially switch that vignette to an explicit test.

Come back to this.